hkmc2/shared/src/test/mlscript/logicsub/MarlowWadler97.mls

```diff
@@ -53,10 +53,8 @@ test(fls, fls)
 
 
 :fixme
-:e
```

Comment:
Instead of removing this `:e`, which should be there, you can make this a `:todo` (instead of `:fixme`)